CorsixTH/Lua/graphics.lua

```diff
@@ -695,15 +695,24 @@ function AnimationManager:setMarkerRaw(anim, fn, arg1, arg2, ...)
   local anim_length = self:getAnimLength(anim)
   local frame = self.anims:getFirstFrame(anim)
 
+  -- Set marker for frame, x1y1 means position is defined in x1 and y1 already
```

Comment:
This doesn't make sense to me. Positions are defined in frames rather than combined x and y positions.

Imho, just make it a proper luadoc.